validation/jinja2/validation/_base.html

```diff
@@ -54,6 +98,16 @@
         <button class="btn btn-outline-success my-2 my-sm-0" type="submit">Search</button>
       </form>
     </div>
+
+    {# Login button #}
+    <div>
+      <a href="/login" style="color: white;">
```

Comment:
Extract inline style as a class?

Reply:
Most definitely. This will also be handled in the styles PR to come.